client/jetpack-onboarding/constants.js

```diff
@@ -28,7 +28,7 @@ export const JETPACK_ONBOARDING_STEPS = {
 };
 
 export const JETPACK_ONBOARDING_SUMMARY_STEPS = {
-	JETPACK_CONNECTION: translate( 'Jetpack Connection' ),
+	JETPACK_CONNECTION: translate( 'Connect to WordPress.com' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Connect to WordPress.com', { context: 'Jetpack Connect post-plugin-activation step, Connect to WordPress.com button'} )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jpo-summary-jpc-link-wording)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).